### ISSUE TITLE
Alibi (AnchorTabular) python interface/method to add to Seldon Graph

### ISSUE THREAD

**Issue by sgarrepalli27** [SeldonIO/alibi#272]
Followed the documentation and I was able to run AnchorTabular locally. Currently, I use `seldon-core` to for real-time model inference built on kubernetes infra. Every node type in Seldon Graph, has a method signature/interface. for eg. **`OUTPUT_TRANSFORMER`** node expects `transform_output` method; Machine Model can be wrapped into python class by including `predict` method. Similarly, I dont see any documentation for Alibi Explainers.

What would be the interface/method to be fulfilled to add Explainer to an existing Seldon graph?
In other words: for the given example, if explainer is a separate python file, what method does seldon orchestrator expects while executing the graph. Reference - https://github.com/SeldonIO/seldon-core/blob/990a565ecd627401f33bd6eac8ef23c22eaa0c87/notebooks/resources/income_explainer.yaml

**Comment by cliveseldon** [comment]
The explainer is separate and not part of the Seldon Graph or handled by the orchestrator. The interface to a trained explainer is a Seldon Core responsibility. You may want to open an issue on that project as Alibi works purely as a python library.
